etl/nixpkgs-graph.nix

```diff
@@ -3,7 +3,13 @@
 
 let
   nixpkgsFlakeRef = builtins.getEnv "NIXPKGS_FLAKE_REF";
-  pkgs = import (builtins.getFlake nixpkgsFlakeRef) { };
+  pkgs = import (builtins.getFlake nixpkgsFlakeRef) {
+    config = {
+      permittedInsecurePackages = [
```

Comment:
Would you mind adding a comment here as to why this is necessary? That way future us will have an easier time if / when we have to revisit this file.

Reply:
I assume that a recent nixos/nixpkgs update made the configuration necessary, even for code that predates this PR.

Reply:
Yep makes sense that if we are evaluating all of nixpkgs we would get some warnings like this. I just meant could you add a comment in this file making a note of why we have to add the whitelist here? Otherwise this might be a bit hard for a future contributor to decipher why we've done this.

Reply:
Oh I see. Yes for sure!